static/app/views/dashboards/widgets/timeSeriesWidget/releaseBubbles/useReleaseBubbles.tsx

```diff
@@ -337,7 +344,7 @@ export function useReleaseBubbles({
   }
 
   return {
-    createReleaseBubbleHighlighter,
+    createReleaseBubbleHighlighter: handleChartRef,
```

Comment:
hmm might be good to rename `createReleaseBubbleHighlighter` to maybe `connectRef`?